src/application/CLI/CommandLineInterface.java

```diff
@@ -33,9 +35,18 @@ public void launchUmlEditor(){
 
         boolean shouldTerminate = false;
         while (!shouldTerminate) {
-            int userChoice = getUserChoice();
+            String userChoice = getUserChoice();
+            if(userChoice.isEmpty()){
+                continue;
+            }
+            if (!Character.isDigit(userChoice.charAt(0))){
+                //autocomplete methods
+                shouldTerminate = AutoCompleteControl(userChoice.trim(), shouldTerminate);
```

Comment:
If you're assigning shouldTerminate to the value that AutoCompleteControl is returning (true in the case of exit being entered), there's no reason to pass it down and modify it in the method

Reply:
Good point, I will remove the boolean passed in.